src/dev/i18n/utils.js

```diff
@@ -303,7 +303,7 @@ export function extractDescriptionValueFromNode(node, messageId) {
 
 export function extractValuesKeysFromNode(node, messageId) {
   if (!isObjectExpression(node)) {
-    throw createFailError(`"values" value should be an object expression ("${messageId}").`);
+    throw createFailError(`"values" value should be an inline object expression ("${messageId}").`);
```

Comment:
Could we perhaps use "inline object literal" since I think object literal might be the more common (and afaik officially correct) term for what we're meaning here?